### ISSUE TITLE
Why create a new list instead of contributing to awesome lists related to fair and RSE already out there?

### ISSUE THREAD

**Issue by broeder-j** [hifis-net/awesome-rse-fair#4]
Was there a special motivation to create another awesome list around these topics? Or did you just miss the others?

For example: 

https://github.com/Materials-Data-Science-and-Informatics/awesome-fair-data
https://github.com/ZoranPandovski/awesome-open-science
...

**Comment by geyslein** [comment]
Thanks for pointing us to these lists. They are really interesting for us. 

- The second one is about open science and not about RSE. I'd rather have a focused list on a specific topic than having a (probably) enormous list about open science, that covers all the different aspects of open science, where *open source* software is just one part of it. (There are awesome lists about OER, Open Access etc. as well....
- The first list touches RSE or rather software here and then. I again miss a relevant focus on RSE.

We'll harvest the lists for suitable information that can be transferred to the RSE awesome lists. Thanks a lot for pointing us to these lists.

To put it short, these RSE awesome lists are not another awesome lists around *these* topics. They are lists around a different topic: RSE.

**Comment by broeder-j** [comment]
Thanks, makes sense.
For me, RSE-FAIR is a sub part of FAIR research in general.
And FAIR is a sub part of open science. OS RSE is a subpart of RSE. So I do not care about the other RSE lists.

Through the first list is named FAIR-data from us is it meant to be around FAIR in general.
Overall I dislike having two FAIR awesome lists within the HGF.
So how about we rename the awesome-fair-data list to awesome-fair, you add the RSE related parts which are missing there and link to that list?

You might also find this list helpful:
https://github.com/INRIA/awesome-open-science-software

**Comment by geyslein** [comment]
HI @broeder-j  we discussed your concern internally at HIFIS. We have set up a neat structure around RSE with the collection of these lists here which we want to keep as specific as it is and focus on RSE. Our goal is to highlight RSE and not to get lost in a sub-category of a huge list. 
This said, we can of course discuss how to avoid redundancy. Where possible we link to other more specific awesome lists, e.g. the [awesome research software repositories ](https://github.com/NLeSC/awesome-research-software-registries). This would be also an opportunity for you to just link to some of 'our', more specific lists.
